src/Workspaces/Core/MSBuild/MSBuild/MSBuildProjectLoader.cs

```diff
@@ -28,8 +28,7 @@ public partial class MSBuildProjectLoader
         private readonly ProjectFileLoaderRegistry _projectFileLoaderRegistry;
 
         // used to protect access to the following mutable state
-        private readonly NonReentrantLock _dataGuard = new NonReentrantLock();
-        private ImmutableDictionary<string, string> _properties;
```

Comment:
Was `_properties` unused?

Reply:
yes it was not read or written to anywhere. included it in this branch by mistake as I was staging/unstaging ranges

Reply:
> src\Workspaces\Core\MSBuild\MSBuild\MSBuildProjectLoader.cs(44,13): error CS0103: The name '_properties' does not exist in the current context

:)